print_performance_metrics.sh

```diff
@@ -1,22 +1,40 @@
 # This script collects metrics like log-per-sec, cpu-percentage, cpu-cores and memory required per iterations in running reg-ex rules
 # in discovering log-levels.
-
+# collector values could be fluentd, fluentbit
+export collector=$1
```

Comment:
looks good to me. Maybe @jcantrill will want to take a quick second look at the changes to this file